.github/workflows/test-pull-request.yml

```diff
@@ -16,45 +19,78 @@ on:
   pull_request:
     branches: [main]
 
+###############
+# Set the Job #
+###############
 jobs:
   validate:
+    name: Lint Code Base
+    # Set the agent to run on
     runs-on: ubuntu-latest
-    steps:
 
-      - name: Checkout repo
+    ##################
+    # Load all steps #
+    ##################
+    steps:
+      ##########################
+      # Checkout the code base #
+      ##########################
+      - name: Checkout Code
         uses: actions/checkout@v3
         with:
           # Full git history is needed to get a proper list of changed files
           # within `super-linter`
           fetch-depth: 0
 
-      - name: Lint files
+      ################################
+      # Run Linters against code base #
+      ################################
+      - name: Lint Code Base
+        #
         # Use full version number to avoid cases when a next
         # released version is buggy
-
         # About slim image: https://github.com/github/super-linter#slim-image
         uses: github/super-linter/slim@v4.9.2
         env:
           VALIDATE_ALL_CODEBASE: false
           DEFAULT_BRANCH: main
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
           VALIDATE_GITHUB_ACTIONS: true
           VALIDATE_GITLEAKS: true
-
+          #
           # The Markdown rules are defined at
           # .github/linters/.markdown-lint.yml
-
+          #
           # Documentation on rules:
           # https://github.com/DavidAnson/markdownlint/blob/main/doc/Rules.md
           VALIDATE_MARKDOWN: true
           VALIDATE_YAML: true
-      
-      - name: Install dependencies
-        uses: bahmutov/npm-install@v1
+
+      - name: Setup Node v16 for Yarn v3
+        uses: actions/setup-node@v3
+        with:
+          node-version: "16.15.0" # Current LTS version
+
+      - name: Enable Corepack for Yarn v3
+        run: corepack enable
+
+      - name: Update to Yarn v3
```

Comment:
Seems like "update" isn't quite accurate. Is it "use" or "install" or something?

Reply:
Install. Fixed.